### ISSUE TITLE
import a blender fbx file

### ISSUE THREAD

**Issue by meah** [mpreble/UnityFBXImporter#7]
when tried to import fbx file by using a file exported from blender i didn't succeeded.

**Comment by alexzpopov** [comment]
use asc (TEXT) format in export

**Comment by meah** [comment]
yes, i already used ASCII but when open the file with notepad i found that the structure of the file is different from autodesk fbx file and that is normal, but in your code it looks like that is the same for both

for example in BlenderConnections = > case "C:" there's no C: in in blender fbx file, it's Connect: instead C:
